### ISSUE TITLE
Checkpoint Result

### ISSUE THREAD

**Issue by cuiyuexin** [yz-cnsdqz/PSI-release#6]
Hello!~

I'm sorry to disturb you but I have a question now. I run 'demo.ipynb' using checkpoint of S1 model pretrained in PROX-E training set with the complete version of loss function. However, when generating people in MPH1Library scene(test scene), I found the generated people usually penetrate the table, sometimes in the middle of the table. I'm not sure whether I made some mistake. And do I need to retrain the model for more than 30 epochs by myself?

Thank you very much!

**Comment by yz-cnsdqz** [comment]
> Hello!~
> 
> I'm sorry to disturb you but I have a question now. I run 'demo.ipynb' using checkpoint of S1 model pretrained in PROX-E training set with the complete version of loss function. However, when generating people in MPH1Library scene(test scene), I found the generated people usually penetrate the table, sometimes in the middle of the table. I'm not sure whether I made some mistake. And do I need to retrain the model for more than 30 epochs by myself?
> 
> Thank you very much!

Thanks for pointing this out. This is one of the biggest limitations of our current method, and we have discussed in the "failure case" section in the manuscript. When the body interpenetrates with a thin structure, the collision loss term and the contact loss term are both very small. This means: 1) interpenetrating with a thin structure is falsely regarded as plausible, and 2) the geometry-aware fitting could not save these failed cases. We need better representations of human-scene interaction. 

Also, as you mentioned, training data is also a problem. If you have better training data than prox, I highly encourage you to try, and tell me the results : )

Currently we are working on a better solution for these issues.

**Comment by cuiyuexin** [comment]
Thanks for your kindly reply! When I load the data in 'realcams.hdf5' and use the visualization method of demo to visualize a body mesh, sometimes the feet will be under the floor. I wonder whether it is due to the SMPLify-X fitting error. And is the body parameters in 'realcams.hdf5' the result of SMPL-X fitting?

**Comment by yz-cnsdqz** [comment]
> Thanks for your kindly reply! When I load the data in 'realcams.hdf5' and use the visualization method of demo to visualize a body mesh, sometimes the feet will be under the floor. I wonder whether it is due to the SMPLify-X fitting error. And is the body parameters in 'realcams.hdf5' the result of SMPL-X fitting?

they are given by PROX.

**Comment by cuiyuexin** [comment]
OK, thanks very much!
